zerver/views/invite.py

```diff
@@ -52,7 +49,7 @@ def invite_users_backend(request, user_profile,
             return json_error(_("Stream does not exist: %s. No invites were sent.") % (stream_name,))
         streams.append(stream)
 
-    do_invite_users(user_profile, invitee_emails, streams, invite_as_admin, body)
+    do_invite_users(user_profile, invitee_emails, streams, invite_as_admin)
```

Comment:
Is this the last remaining callsite using that "custom body" parameter to `do_invite_users`? (If not, we should remove any others too.)

Then let's remove that parameter from the function itself, too.

Reply:
Also follow the trail of what `do_invite_users` does with the data -- you'll want to go to the code in `zerver/worker/queue_processors.py` that processes the `invites` queue.

Reply:
1. Removed parameter from function too.
2. data was used as context in `zerver/worker/queue_processors.py`. I removed line containing `body`.